src/robusta/core/discovery/discovery.py

```diff
@@ -87,7 +88,29 @@ class Config:
 
 DISCOVERY_STACKTRACE_FILE = "/tmp/make_discovery_stacktrace"
 DISCOVERY_STACKTRACE_TIMEOUT_S = int(os.environ.get("DISCOVERY_STACKTRACE_TIMEOUT_S", 10))
-
+KIND_TO_COREV1_METHOD = {
+    "pods": "list_pod_for_all_namespaces",
+    "configmaps": "list_config_map_for_all_namespaces",
+    "endpoints": "list_endpoints_for_all_namespaces",
+    "services": "list_service_for_all_namespaces",
+    "secrets": "list_secret_for_all_namespaces",
+    "persistentvolumeclaims": "list_persistent_volume_claim_for_all_namespaces",
+    "serviceaccounts": "list_service_account_for_all_namespaces",
+    "replicationcontrollers": "list_replication_controller_for_all_namespaces",
+    "limitranges": "list_limit_range_for_all_namespaces",
+    "resourcequotas": "list_resource_quota_for_all_namespaces",
+    "events": "list_event_for_all_namespaces",
+    "podtemplates": "list_pod_template_for_all_namespaces",
+    # Cluster scoped
```

Comment:
Do we really need those?
Because we don't store anything that isn't namespace scoped